IoTConnect_2.0/IoTConnect_2.0/IoTConnectManager/IoTConnectService.swift

```diff
@@ -1913,5 +1949,16 @@ extension IoTConnectManager {
                 print("Error parsing EdgeRule Response")
             }
         }
-    } 
+//<<<<<<< Updated upstream
```

Comment:
remove unwanted comments

Reply:
@sw-ot-deveshmevada Changes are done in new push